discord-readme.md

```diff
@@ -41,6 +41,8 @@ TEXT CHANNELS: These consist of the server's main text channels, that are for di
 
 -> #standup: everyday of code, give us your standup updates! What you did yesterday,what you plan to do today and any blockers?
 
+-> #healthfitness: health and fitness caus you want to get those bizeps

```

Comment:
```suggestion
-> #healthfitness: health and fitness cause you want to get those biceps

```

Reply:
Actually a better sentence would be " health and fitness so you can get those biceps"

Reply:
Yep will change it